bitbots_dynup/include/bitbots_dynup/dynup_stabilizer.h

```diff
@@ -4,30 +4,37 @@
 #include <optional>
 #include <moveit/robot_state/robot_state.h>
 #include <moveit/robot_model_loader/robot_model_loader.h>
-#include <geometry_msgs/Pose.h>
+#include <sensor_msgs/Imu.h>
 #include <tf2_ros/transform_listener.h>
 #include <bitbots_splines/abstract_stabilizer.h>
 #include "dynup_utils.h"
+#include <moveit/robot_state/robot_state.h>
+#include <tf2_ros/transform_listener.h>
+#include <control_toolbox/pid.h>
 
 namespace bitbots_dynup {
 
 class Stabilizer : public bitbots_splines::AbstractStabilizer<DynupResponse> {
  public:
   void init(moveit::core::RobotModelPtr kinematic_model);
   DynupResponse stabilize(const DynupResponse &response, const ros::Duration &dt) override;
+  void setTransforms(geometry_msgs::TransformStamped to_trunk);
   void useStabilizing(bool use);
-  void useMinimalDisplacement(bool use);
-  void setStabilizingWeight(double weight);
+  void setRobotModel(moveit::core::RobotModelPtr model); 
   void reset() override;
- private:
-  robot_state::RobotStatePtr goal_state_;
+  sensor_msgs::Imu imu_;
+  control_toolbox::Pid pid_trunk_pitch_;
+  control_toolbox::Pid pid_trunk_roll_;
 
+
+private:
+  robot_state::RobotStatePtr goal_state_;
   robot_model::RobotModelPtr kinematic_model_;
-  moveit::core::JointModelGroup *legs_joints_group_;
+  geometry_msgs::TransformStamped to_trunk_;
```

Comment:
Please add a comment. What is this needed for? From what to trunk? (r_sole, apparently)
Preferably also for the `setTransforms` method.